frontend/src/components/RecommendationList.jsx

```diff
@@ -1,6 +1,28 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Link, useLocation } from "react-router-dom";
+import axios from "axios";
+
+// comment out the 2 console logs to get a glimpse of what's in there :)
 
 const RecommendationList = () => {
+  const [myData, setMyData] = useState(null);
+  const navigationLocation = useLocation();
+  // console.log(navigationLocation);
+
+  // forward the values of the form submission and send a get request with them
+  // localhost will be changed, just here while in development
+  const getData = async () => {
+    const response = await axios.get("http://localhost:8080/api/wines", {
+      params: navigationLocation.state,
+    });
+    setMyData(response.data);
+    // console.log(response.data);
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
```

Comment:
Better move the `getData` function into inside the effect: https://overreacted.io/a-complete-guide-to-useeffect/#moving-functions-inside-effects
Also, using navigation state is pretty rare actually, usually the best approach is to use query parameters in the url, this way you can share the url with anyone and they will see the same the page you shared. But we can tackle this in another PR.